CMake/SENSEIConfig.cmake.in

```diff
@@ -60,7 +61,7 @@ elseif (ENABLE_VTK_CORE)
   if (NOT VTK_DIR)
     set(VTK_DIR "@VTK_DIR@")
   endif()
-  find_dependency(vtk)
+  find_dependency(VTK)
```

Comment:
OK

Reply:
This one gave a strange error, it finds the VTK config, but inside of `vtk-config.cmake` the include of `${CMAKE_FIND_PACKAGE_NAME}-targets.cmake` fails because the package name is `vtk` instead of `VTK`. First time I have run into that!